commands/Fun/say.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2017-2018 dirigeants. All rights reserved. MIT license.
+const { Command } = require("klasa");
+
+module.exports = class extends Command {
+  constructor(...args) {
+    super(...args, {
+      description: "Make the bot say something",
+      name: "say",
+      usage: "<message:string>",
+      disableEveryone: true,
+      permissionLevel: 4,
+      aliases: ["echo"]
+    });
+  }
+
+  async run(msg, [args]) {
+    msg.delete().catch() => {} 
```

Comment:
This is not what I meant... This isn't even valid js

Reply:
You told me to replace (O_o=>{} to () => {}

Reply:
> I love the `O_o => {}`, but maybe change it to ` () => {}`

Attention to detail. And he meant the contents of the .catch

Reply:
> > I love the `O_o => {}`, but maybe change it to ` () => {}`
> 
> Attention to detail. And he meant the contents of the .catch

Yeah, i just fixed & pushed it.